### PR TITLE
3. Apply black auto-code formater

### PR DESCRIPTION
**Pull request by amakhaba**
This PR does the following ([Apply black](https://github.com/ska-sa/katdeveloper/wiki/Investigate-a-way-to-improve-Py3-migration-feedback-loop-on-tox#auto-code-format)):
- Applies black auto-code formater on the utilities folder

*Definition of Done Checklist*

- [x] Code meets our [python style guidelines](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit?usp=sharing)?
- [x] Unit tested (coded, passed, included)?
- [x] Requested at least 2 reviewers?
JIRA: [MT-738](https://skaafrica.atlassian.net/browse/MT-738)

### CODE REVIEW

**Comment by mmphego** [comment]
Why so many PR? 
Is the a sequence you are following, wrt merging to master?

**Comment by tockards** [comment]
@mmphego The sequence we are trying to follow is that the PRs cascade.
You should review them in order from the earliest one first and merge them in that way also.
This is my interpretation of 
> The idea is to create small incremental PR's and deliver value instead of one giant intimidating PR.

We wanted to carry on yesterday as there were not many people reviewing the PRs and it was blocking us from carrying on. 



Knowing your love for blog posts you can read these below.

 https://graysonkoonce.com/stacked-pull-requests-keeping-github-diffs-small/ 
https://github.com/frappe/erpnext/wiki/Cascading-Pull-Requests

File: tango_simlib/utilities/testutils.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from __future__ import print_function
 from __future__ import division
 from __future__ import absolute_import
```
Comment (mmphego) [inline tango_simlib/utilities/testutils.py]: Please place the imports below the licensing information.

**Comment by mmphego** [comment]
PR is blocked by #106

File: tango_simlib/utilities/testutils.py (hunk 1 above)
Comment (mmphego) [reply tango_simlib/utilities/testutils.py]: PR #108 resolves this.

File: tango_simlib/tango_launcher.py [hunk 2]
```diff
@@ -21,41 +21,65 @@
...
     dev_info.name = name
     dev_info._class = device_class
-    dev_info.server = "{}/{}".format(server_name.split('.')[0], instance)
-    print("""Attempting to register TANGO device {!r}
+    dev_info.server = "{}/{}".format(server_name.split(".")[0], instance)
+    print(
+        """Attempting to register TANGO device {!r}
     class: {!r}  server: {!r}.""".format(
-        dev_info.name, dev_info._class, dev_info.server))
+            dev_info.name, dev_info._class, dev_info.server
```
Comment (lanceWilliams) [inline tango_simlib/tango_launcher.py]: The indenting here looks weird. Does the linter not complain?

Comment (mmphego) [reply tango_simlib/tango_launcher.py]: Black did the linting.
No complaints from flake8

File: tango_simlib/tests/test_simdd_json_parser.py [hunk 3]
```diff
@@ -626,52 +784,58 @@ def test_xmi_simdd_attribute_parameters_when_both_specified(self):
         those of the simdd override the ones in xmi in the configured model
 
         """
-        attr_with_overrriden_info = 'temperature'
+        attr_with_overrriden_info = "temperature"
         simdd_specified_temperature_attr_params = {
-                'description': 'Current actual '
-                'temperature outside near the telescope.',
-                'min_value': '-15', 'max_value': '55'}
+            "description": "Current actual " "temperature outside near the telescope.",
```
Comment (lanceWilliams) [inline tango_simlib/tests/test_simdd_json_parser.py]: Are we fixing these by hand or leaving them as is?

**Pull request merged**